packages/loot-core/src/client/query-hooks.ts

```diff
@@ -0,0 +1,53 @@
+import { useState, useMemo, useEffect, type DependencyList } from 'react';
```

Comment:
This is the same file as `query-hooks.tsx`. Except: removed `makeContext` (no longer used) and fixed some small strict TS issues (thus making this file strict TS).